src/components/AssetMeta/AssetMeta.tsx

```diff
@@ -247,7 +277,8 @@ const AssetMetaHeader = styled.div<{ isLoading: boolean }>`
   visibility: ${({ isLoading }) => (isLoading ? 'hidden' : 'visible')};
 `;
 
-const Component = withDefaultTheme(AssetMeta);
-Component.displayName = 'AssetMeta';
+const ComponentWithTheme = withDefaultTheme(AssetMeta);
+ComponentWithTheme.displayName = 'AssetMeta';
 
-export { Component as AssetMeta };
+export { AssetMeta as AssetMetaWithoutTheme };
+export { ComponentWithTheme as AssetMeta };
```

Comment:
but we didn't export `AssetMetaWithoutTheme ` before, does anyone need it?

Reply:
@polomani we want this for showing props documentation.

Reply:
yes, but be careful here, we need to have ability to use only internal, we shouldn't expose `AssetMetaWithoutTheme` outside the library. To avoid this you need to add only exposed components and interfaces in the export section of `index.ts`